store/commitment/store.go

```diff
@@ -411,7 +413,6 @@ loop:
 			if err != nil {
 				return snapshotstypes.SnapshotItem{}, fmt.Errorf("failed to import tree for version %d: %w", version, err)
 			}
-			defer importer.Close()
```

Comment:
You don't have an option for this code than to leave it as is, because it got created as is and for all we know gets replaced in another loop iteration so please keep this code.